packages/freezed/lib/src/templates/properties.dart

```diff
@@ -9,6 +9,16 @@ import '../utils.dart';
 import 'concrete_template.dart';
 import 'prototypes.dart';
 
+const privConstUsedErrorString =
+    'It seems like you constructed your class using MyClass._()'
+    ' or by manually adding another constructor. '
```

Comment:
It isn't possible to add other constructors, so this mention is not necessary

Reply:
Alright, removing it tonight.